shortcodes/ucf-faq-list-shortcode.php

```diff
@@ -87,7 +87,7 @@ public static function shortcode( $atts ) {
 			if ( $atts['tags'] !== '' ) {
 
 				$related_posts = UCF_FAQ_Common::get_related_faqs( $atts['tags'], $items );
-				$related_faqs_html = '<' . $atts['related_element'] . ' class="' . $atts['related_class'] . '">Related FAQs</' . $atts['related_element'] . '>';
+				$related_faqs_html = '<' . $atts['related_element'] . ' class="pt-4 ' . $atts['related_class'] . '">Related FAQs</' . $atts['related_element'] . '>';
```

Comment:
I think increasing this to `pt-5` helps define more separation between related and topic questions